starcheck/src/starcheck.pl

```diff
@@ -211,6 +219,7 @@
 
 
 # First read the Backstop file, and split into components
+print "Reading backstop file $backstop\n";
```

Comment:
Are these just added because you wanted more logging?

Reply:
It was more about profiling / timing of different steps. But yes, I never like it when starcheck just hangs for a while without any sign of activity. Early on reading the various files was taking a long time... You can cut these if you want, in the current version the period of inactivity is not *too* bad.